src/EnergyPlus/SolarShading.cc

```diff
@@ -7194,6 +7194,10 @@ namespace SolarShading {
 			} else {
 				SUN3( DayOfYear, AvgSinSolarDeclin, AvgEqOfTime );
 				AvgCosSolarDeclin = std::sqrt( 1.0 - pow_2( AvgSinSolarDeclin ) );
+				// trigger display of progress in the simulation every two weeks
```

Comment:
Seems like a reasonable location to me, but are there side effects of setting that flag to true here?  Does it ever get set back to false?  Does it need to?